README.md

```diff
@@ -65,6 +65,12 @@ sudo python3 setup.py install
 
 This should install all the dependencies required to run PyRDP.
 
+### Installing with Docker-compose
+PyRDP can be installed in a container using Docker-compose with this command :
+```
+sudo docker-compose up
+```
+
```

Comment:
I asked for the docker to be documented, not docker-compose. The options should be explained: volumes to gather log files and arguments needed to launch graphical interface (DISPLAY, etc.)